spec/javascripts/AccessTokens/components/ExpirationDatePicker.spec.tsx

```diff
@@ -86,7 +86,7 @@ describe('select a period', () => {
     const expectedInputValue = moment(inputDate).utc().format(TIMESTAMP_FORMAT)
 
     expect(inputValue).toEqual(expectedInputValue)
-    expect(inputDate).toBeWithinSecondsFrom(targetDate)
+    expect(inputDate).toBeWithinSecondsFrom(targetDate, 5)
```

Comment:
Based on @jlledom 's comment: https://github.com/3scale/porta/pull/4189#pullrequestreview-3578333521

But I decided to be on the safe side and add a 5 secs margin - it doesn't matter much whether it's 2 or 5. The selectors are actually days-based, the timestamp is visible at a minute accuracy (e.g. `The token will expire on March 30, 2026 at 12:42 GMT+2`), so I think 5-sec margin is completely fine and should eliminate false negatives.